docusaurus.config.js

```diff
@@ -245,14 +245,20 @@ const config = {
     [
       '@docusaurus/plugin-client-redirects',
       {
+        redirects: [
+          {
+            to: '/docs/references/tenants/#tenant-type',
+            from: ['/docs/recipes/tenant-type', '/docs/references/tenant-type'],
+          },
+        ],
         // `existingPath` is the path `to`, the return value is the path `from`.
         createRedirects(existingPath) {
           if (existingPath.includes('/quick-starts')) {
             return existingPath.replace('/quick-starts', '/sdk');
           }
 
-          if (['/references/openid-connect', '/references/tenant-type'].some((path) => existingPath.includes(path))) {
-            return existingPath.replace('/references/', '/recipes/');
+          if (existingPath.includes('/references/openid-connect')) {
+            return existingPath.replace('/references/openid-connect', '/recipes/openid-connect');
```

Comment:
Can this be written as the simple "from" & "to" rule?

Reply:
no, `openid-connect` has several pages